src/renderers/PartialViewRenderers/region/facilitiesInfo.js

```diff
@@ -7,14 +7,27 @@ $(function () {
     addFacility_handler();
     //handle delete facility
```

Comment:
Remove this comment too, I suggest refactoring this entire string of function calls. Functions should always be verb phrases. These are nouns

Reply:
Alright